marklogic-data-hub/src/test/java/com/marklogic/hub/HubTestBase.java

```diff
@@ -705,6 +707,22 @@ protected int getDocCountByQuery(String database, String query) {
         return count;
     }
 
+    protected JSONObject getQueryResults(String query, String database) {
```

Comment:
Can this just return a JsonNode instead of JSONObject? What's the advantage of JSONObject over JsonNode?